72pe.typ

```diff
@@ -168,9 +261,130 @@
 // -----------------------------------------------------------------------------
 // Ground Checklists and Information page
 // -----------------------------------------------------------------------------
-#let ground_checklists_and_info = [
+#let ground_checklists_and_info_columns = [
+	#checklist("Preflight", palette.brown,
+		checklist_group("Fluids"),
+		([Engine master], [OFF]),
+		([Fuel quantity], [CHECK VISUALLY]),
+		([Fuel sumps], [DRAIN],
+		 [Left wing, right wing, fuel strainer. Inspect for contamination.]),
+		([Fuel filler caps], [SECURE]),
+		([Engine oil level], [CHECK], [Minimum 6 quarts]),
+		([Gearbox oil level], [CHECK], [Oil must cover at least 1/2 of inspection glass]),
+		checklist_group("Cabin"),
+		([Pitot cover], [REMOVE]),
+		([POH], [VERIFY PRESENT]),
+		([Master], [ON]),
+		([Flaps], [EXTEND]),
+		([Fuel gauges], [CHECK QUANTITY]),
+		([Water level light], [CHECK OFF]),
+		([Avionics], [ON], [Check avionics fan audible]),
+		([Avionics], [OFF]),
+		([Annunciator panel], [TST]),
+		([Lights], [CHECK]),
+		([Pitot heat], [TEST]),
+		([Master], [OFF]),
+		([Fire extinguisher], [SECURE]),
+		([Control lock], [REMOVE]),
+		checklist_group("Empennage"),
+		([Rudder gust lock], [REMOVE]),
+		([Control surfaces], [CHECK]),
+		checklist_group("Right"),
+		([Main wheel tire], [CHECK INFLATION]),
+		([Aileron], [CHECK]),
+		checklist_group("Nose"),
+		([Propeller, spinner], [CHECK]),
+		([Air filter], [CHECK CLEAR]),
+		([Nosewheel & strut], [CHECK INFLATED]),
+		([Static source], [CHECK CLEAR]),
+	)
+	#checklist("Preflight (continued)", palette.brown,
+		checklist_group("Left"),
+		([Main wheel tire], [CHECK INFLATION]),
+		([Fuel tank vent], [CHECK CLEAR]),
+		([Pitot tube], [CHECK CLEAR]),
+		([Stall warning], [TEST]),
+		([Aileron], [CHECK]),
+		checklist_group("Final"),
+		([Flight Circle], [DISPATCH]),
+		([Hobbs times], [RECORD]),
+		([Baggage door], [LOCK]),
+		([Chocks], [REMOVE]),
+		([Tie-downs], [REMOVE]),
+	)
+	#checklist("Securing", palette.black,
+		([Control lock], [INSTALL]),
+		([Tie-downs, chocks], [APPLY]),
+		([Vents, windows], [CLOSE]),
+		([Pitot cover], [APPLY]),
+		([Flight Circle], [CHECK IN]),
+		([Doors], [LOCK]),
+	)
+	#container("Speeds", palette.purple, inset: false, {
+		set par(leading: 0.3em)
+		table(
+			align: (left + horizon, center, center),
+			columns: (auto, 1fr, auto),
+			fill: (_, row) => (none, palette.grey).at(calc.rem(row, 2)),
+			inset: (x: title_inset, y: .3em),
+			stroke: none,
+			[], [], [KIAS],
+			table.hline(stroke: .05em + palette.purple),
+			[Best glide (V#sub[G])], [], [65],
+			[Best angle\ of climb (V#sub[X])], [Sea level\ 10,000 ft], [60\ 65],
+			[Best rate\ of climb (V#sub[Y])], [Sea level\ 10,000 ft], [79\ 71],
+			[Landing approach], box(align(left)[Flaps up\ Flaps 40]),
+				[69-80\ 60-70],
+			[Normal takeoff\ climb], [], [70-80],
+			[Short-field\ takeoff climb], [Flaps 10], [57],
+			[Normal enroute\ climb], [Sea level\ 10,000 ft], [75-85\ 70-80],
```

Comment:
Where does this come from? Page 4-25 of the AFMS suggests 75-80 KIAS with no altitude dependence.